### ISSUE TITLE
nsample missing arguments

### ISSUE THREAD

**Issue by eric-le-12** [kaanakan/slamp#2]
Hi during running your code, i was unexpectedly see an error due to missing arguments

File "/notebooks/slamp/helpers.py", line 362, in eval_step
    nsample = opt.nsample

File args.py doesnt have any definition about nsample, what does nsample mean?
I suppose it should be the number of samples per batch in evaluation which means eval batch size
Thanks for your reading

**Comment by kaanakan** [comment]
Hi,

Thank you for your interest in our work. In the literature, the stochastic models generate more than one sample because the samples differ because of the latent variables. `nsample` refers to the number of samples to generate for each example in the test/validation set. 

Since it is used in the evaluation code, the evaluation code contains that `nsample` argument. 

https://github.com/kaanakan/slamp/blob/7c47e6b561d1f973626061d8e8600e072ed6785e/evaluate.py#L67

However, if you encounter that error while training the model, then we need to fix that issue by adding `nsample` to args.py.

Best regards,
Kaan
